### ISSUE TITLE
Cannot establish a connection to BeEF User Interface on my browser

### ISSUE THREAD

**Issue by codemania254** [beefproject/beef#1743]
Hello guys, 

I am new to Kali Linux and BeEF among other tools. I have setup BeEF in Kali Linux which is running on VM Virtual Box (windows 10). I started beEF with the usual commands  i.e cd /usr/share/beef-xss
./beef

The problem I am facing is that when I copy-paste the beEF user interface urls on any of my browsers so I can login with default credentials for BeEF, the page does not load. The browser tells me I cannot connect to the server. I tried 2 urls http://127.0.0.1:3000/ui/panel and http://0.0.0.0:3000/ui/panel and they are not loading. How do I go about it. My goal is to load the BeEF login page with famous blue cow image. Please explain it the way you would do it to a newbie. Thanks

**Comment by bcoles** [comment]
`127.0.0.1` and `0.0.0.0` are not the URLs you're looking for:

* https://en.wikipedia.org/wiki/Localhost
* https://en.wikipedia.org/wiki/0.0.0.0

**Comment by bcoles** [comment]
You can find the IP address of your system using the `ifconfig` or `ip addr` commands.

**Comment by codemania254** [comment]
Bcoles, I am using the IP address of my system which is one of the links that BeEF gave me to log into the user interface on my browser. I have also used ifconfig command to confirm and the IP is the same as what BeEF gave me in the link to the user interface.  No luck so far. I replaced the zeros here http://0.0.0.0:3000/ui/panel with the actual IP address of my system (didn't want to reveal the IP). Any suggestions?
